src/styles/pages/ConnectionPage.css

```diff
@@ -0,0 +1,266 @@
+body, html {
```

Comment:
Let's use bootstrap for styling. Try and replace the styling with bootstrap styling and make is basic as possible for our mvp